src/constructUrl.js

```diff
@@ -1,6 +1,6 @@
 /*
 Copyright © 2015 by Coursera
-Licensed under the Apache License 2.0, seen https://github.com/coursera/react-imgix/blob/master/LICENSE
+Licensed under the Apache License 2.0, seen https://github.com/coursera/react-imgix/blob/main/LICENSE
```

Comment:
Looks like this one should be reverted

Reply:
Oh you're right - I actually picked this up locally but didn't push my commit. Fixing now.